common/verdeps/helpers_test.go

```diff
@@ -0,0 +1,94 @@
+package verdeps
+
+import (
+	"testing"
+	"regexp"
+	"github.com/stretchr/testify/assert"
+)
+/*
+ Untested functions: 
+ isSubPackage (too simple)
+ importPathHashOf (too simple and I don't understand why it's called that)
+ subDirExists (unused) 
+ TODO: getPackageDirPaths have to mock OS.stat 
+*/
+var validInternalDirnameRegex = regexp.MustCompile(`\b[0-9a-f]{16}\b`)
+var generatedInternalDirName string = generateInternalDirName();
+
+const author = "raymondChandler"
+const repo = "theLongGoodBye"
+const subpath = "/goodRead"
+const sixCharSha = "abcdef"
+
+
+func TestParseImportPath_validAuthor(t *testing.T) {
+	t.Parallel()
+	expectedAuthor, expectedRepo, expectedSubpath := author, "", "";
+	actualAuthor, actualRepo, actualSubpath := parseImportPath(githubPrefix + author + "/")
+	assert.Equal(t, expectedAuthor, actualAuthor)
+	assert.Equal(t, expectedRepo, actualRepo)
+	assert.Equal(t, expectedSubpath, actualSubpath)
+}
+
+func TestParseImportPath_invalidAuthor(t *testing.T) {
```

Comment:
I'm guessing the inputs to these functions are cleaned beforehand, because I doubt this is a valid/intended test case.